static/app/views/starfish/components/samplesTable/spanSamplesTable.tsx

```diff
@@ -67,42 +86,56 @@ export function SpanSamplesTable({isLoading, data, p95}: Props) {
   }
 
   function renderBodyCell(column: GridColumnHeader, row: SpanTableRow): React.ReactNode {
```

Comment:
Passing these `commonProps` to every cell is definitely a bummer! I think `GridEditable` should support passing hover handlers to the grid row. Maybe something to bring up in the channel, or TSC meeting on Thursday?

Reply:
Yes, 100% a bummer and I definitely would want to have some sort of row event handlers! I think i'll bring it up the the team/TSC and work on it post release. I wanted to do get this out quicker for the wen release.